### ISSUE TITLE
Epic: provide one or multiple ways to add events to boudicca for event organizersx without website

### ISSUE THREAD

**Issue by twatzl** [boudicca-events/boudicca.events#572]
**Problem Statement**

Many, especially smaller event organizers or event organizers which only host a few events a year either

- don't have a website
- post event announcements on social media rather than on their website
- post events on their website in a format that cannot be read by boudicca (e.g. image or pdf or strangely formatted html)

Currently the events of those organizers are completely excluded. The hurdle for creating a website is quite big for event organizers. Thus a whole bunch of small organizers are currently excluded.

**Requirements**

- As Bill the Boudicca server operator I want to control who can publish events
- As Bob the Boudicca server operator I DO NOT want to have to manually manage events
- As Evan the event organizer I want to create information about my event
- As Ethan the event organizaer I might need to change information about my event
- As Bernhard the Boudicca server operator I might want to provide a structure to Eugen the even organizer, so that he fills out the information I need for my server.

Technical needs:

- It must be possible to create events
- It must also be possible to change or delete events (e.g. due to cancellation or more information gets available)
- It must be possible to add images. (png, jpg) for a preview, but also there might be additional info available
- PDF might be nice (sometimes there is information sheets with additional information)
- It would be cool if the input form is structured, so the data can be accordingly used.
- The server operator should be able to modify the structure before handing it over to event organizers.
- No data is stored on boudicca servers.

**Potential Solutions**

**Google Calendar: create a shared google calendar and scrape that**

Pros:
+ easy to implement
+ easy to use

Cons:
- no structure
- images?

**Google Forms/Sheets?** 

**GitHub Pages + Github repo: Create a form as github pages**

Basically create a github page with a form and on each form submit a github action is triggered that updates a repo and writes the info to it.
